pkg/anonymization/anonymizer.go

```diff
@@ -83,7 +84,7 @@ type ConfigProvider interface {
 
 // NewAnonymizer creates a new instance of anonymizer with a provided config observer and sensitive data
 func NewAnonymizer(clusterBaseDomain string, networks []string, secretsClient corev1client.SecretInterface) (*Anonymizer, error) {
-	networks = append(networks, "127.0.0.1/8")
+	networks = append(networks, "127.0.0.0/8")
```

Comment:
Why this change please? It's not clear to me at the first glance. We need to update also in the package doc - https://github.com/openshift/insights-operator/pull/446/files#diff-9180eb430c735bed4f79b7892293543fe58bd8bc57493f4f823ca84faa073f8eR12

Reply:
It was a mistake in the original code, not really a problem cuz anonymizations would still work correctly, but /8 means everything after the first byte(127) should be zeros so that it's a valid mask.

Reply:
Ahh I see. Thx